firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+WebViewDelegates.swift

```diff
@@ -789,13 +789,32 @@ extension BrowserViewController: WKNavigationDelegate {
         }
 
         if let url = error.userInfo[NSURLErrorFailingURLErrorKey] as? URL {
+            guard var errorURLpath = URLComponents(string: "\(InternalURL.baseUrl)/\(ErrorPageHandler.path)" ) else { return }
+            errorURLpath.queryItems = [URLQueryItem(
+                name: InternalURL.Param.url.rawValue,
+                value: url.absoluteString
+            ), URLQueryItem(
+                name: "code",
+                value: String(
+                    error.code
+                )
+            )]
```

Comment:
```suggestion
           errorURLpath.queryItems = [
                URLQueryItem(
                    name: InternalURL.Param.url.rawValue,
                    value: url.absoluteString
                ),
                URLQueryItem(
                    name: "code",
                    value: String(
                        error.code
                    )
                )
            ]
```

For arrays, suggested spacing is above. It's easier to read if each item is given it's own new line, with the wrapping `[` and `]` on their own lines.